trlx/data/configs.py

```diff
@@ -36,13 +36,13 @@ class ModelConfig:
 
     :param delta_kwargs: Keyword arguments for instantiating OpenDelta models for delta-tuning.
         Follow the `OpenDelta.AutoDeltaConfig` specification, e.g. for LoRA style tuning, set
-        the `delta_type` to `lora` and include the model specific hyper-parameters (e.g. `lora_a`)
-            {"delta_type": "lora", "modified_modules": "all", "lora_a": 0.5}
+        the `delta_type` to `lora` and include the model specific hyper-parameters (e.g. `lora_r`)
+            {"delta_type": "lora", "modified_modules": "all", "lora_r": 5}
```

Comment:
As you observed, `lora_a` should be `lora_alpha` here. I'm okay using the rank hparam in the docs instead though.